packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -303,7 +317,9 @@ export function usePress(props: PressHookProps): PressResult {
               }
             };
 
-            addGlobalListener(getOwnerDocument(e.currentTarget), 'keyup', chain(pressUp, onKeyUp), true);
+            const ownerDocument = getRootNode(e.currentTarget) || getOwnerDocument(e.currentTarget);
```

Comment:
isn't this equivalent?
```
const ownerDocument = getRootNode(e.currentTarget);
```
or when does it fallback to getOwnerDocument?

Reply:
I removed the fallback, it will never fallback as long as there is a `e.currentTarget`.